hbook/ntup/ntup_example_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 The go-hep Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ntup
+
+import (
+	"fmt"
+	"log"
+	"go-hep.org/x/hep/hbook"
+	
+)
+var (
+	nt *Ntuple
+)
+
+func ExampleNtuple_scanH2D() {
+	const nxpoints = 10000
+	nt, err := ntcsv.Open(
```

Comment:
@sbinet, I am not able to access `ntcsv` here. If I do `hbook.ntup.ntcsv.ntcsv.Open` It says `cannot refer to unexported name`. If I import `go-hep.org/x/hep/hbook/ntup/ntcsv` It throws `import cycle error`. Also, If I place this file inside `ntup/ntcsv`. I am not able to access both `hbook` and `ntup` at the same time(If I import `go-hep.org/x/hep/hbook/`, and try to access ntup, it throws `export error`. I am a little dicey about where this file should go. Can you please help me with that?

Reply:
you have to import `"go-hep.org/x/hep/hbook/ntup/ntcsv"` before being able to use `ntcsv.Open`.
if you put this example in the `ntup_test` package (instead of the `ntup` package as is currently done), you'll be able to break the import cycle (`ntcsv` import `ntup`, so `ntup` can't import `ntcsv`.)

see my comment above, at line 5.